girder/utility/path.py

```diff
@@ -44,7 +44,7 @@ def encode(token):
 
 
 def decode(token):
-    """Unescape special characters in a token from a path representation.
+    """Un-escape special characters in a token from a path representation.
```

Comment:
Do we not have a rule for docstrings? Almost all of Girder conforms to putting the subject on its own line, after the triple quotes.